loris/resolver.py

```diff
@@ -60,12 +60,20 @@ class SimpleFSResolver(_AbstractResolver):
 
     def __init__(self, config):
         super(SimpleFSResolver, self).__init__(config)
-        self.cache_root = self.config['src_img_root']
+        if 'src_img_roots' in self.config:
+            self.cache_roots = self.config['src_img_roots']
+        else:
+            self.cache_roots = [self.config['src_img_root']]
 
-    def is_resolvable(self, ident):
+    def get_fp(self, ident):
         ident = unquote(ident)
-        fp = join(self.cache_root, ident)
-        return exists(fp)
+        for directory in self.cache_roots:
```

Comment:
Could you find a way to avoid the loop in the simple case (i.e. one cache root?). Loops are notoriously slow in Python, which means you should consider writing the loop as a list comprehension instead.

Additionally, this should be documented in the config file (and point out that the first match wins, in case of a naming collision), and I'd like to see tests that prove that both cases (single root and multiple) can work.